src/Essentials/src/Geolocation/Geolocation.shared.cs

```diff
@@ -2,26 +2,55 @@
 using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
+using System.ComponentModel;
+using Microsoft.Maui.Essentials;
+using Microsoft.Maui.Essentials.Implementations;
+
 
 namespace Microsoft.Maui.Essentials
 {
+	public interface IGeolocation
+	{
+		Task<Location> LastKnownLocationAsync();
```

Comment:
Can you add the `Get` back to make it `GetLastKnownLocationAsync()` ? Same with the other methods here.